specification/indices/simulate_index_template/IndicesSimulateIndexTemplateRequest.ts

```diff
@@ -32,11 +40,31 @@ export interface Request extends RequestBase {
     /** Index or template name to simulate */
     name: Name
   }
+  query_parameters: {
+    /**
+     * If `true`, the template passed in the body is only used if no existing
+     * templates match the same index patterns. If `false`, the simulation uses
+     * the template with the highest priority. Note that the template is not
+     * permanently added or updated in either case; it is only used for the
+     * simulation.
+     * @server_default false
+     * */
+    create?: boolean
+    /**
+     * Period to wait for a connection to the master node. If no response is received
+     * before the timeout expires, the request fails and returns an error.
+     * @server_default 30s
+     */
+    master_timeout?: Time
+  }
   body: {
-    index_patterns?: IndexName[]
+    index_patterns?: Indices
     composed_of?: Name[]
-    /** Any overlapping templates that would have matched, but have lower priority */
-    overlapping?: OverlappingIndexTemplate[]
```

Comment:
`overlapping` isn't documented but is a part of the response. I couldn't find any mention of it in the Elasticsearch repository either. Perhaps this was a mistake and should have been on the response?

Reply:
@swallez can you double check this?

Reply:
Confirmed. Request body is at https://github.com/elastic/elasticsearch/blob/main/server/src/main/java/org/elasticsearch/cluster/metadata/ComposableIndexTemplate.java

We can also see an `allow_auto_create` optional boolean there.